functions.php

```diff
@@ -598,10 +598,19 @@ function edit_searchwp_query_orderby( $orderby ) {
 	public function populate_act_page_children_options() {
 
 		// Get the id of the ACT page. We need this to get the children posts/pages of the ACT Page.
-		$arguments = [
-			'post_type'     => 'page',
-			'post_name__in' => [ 'act', 'ACT', 'Act' ],
-		];
+		$parent_act_id = planet4_get_option( 'select_act_page' );
+		if( 0 != $parent_act_id ) {
+			$act_page_object  = get_post( $parent_act_id );
+			$arguments = [
+				'post_type'     => 'page',
+				'post_name__in' => [ $act_page->post_name ],
```

Comment:
@sagarsdeshmukh did you mean to use $act_page_object here?

Reply:
Yes I mean the $act_page_object variable. I fixed it.